content/ru/guides/concepts/views.md

```diff
@@ -0,0 +1,221 @@
+---
+title: Views
+description: The Views section describes all you need to know to configure data and views for a specific route in your Nuxt.js Application. Views consist of an app template, a layout, and the actual page.
```

Comment:
Nice, can you also translate the description and the quiz. the quiz is not used right now but it will be soon. thanks